.github/workflows/build_frontend.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
           list-files: csv
           filters: |
             frontend:
-              - 'frontend-react/**'
+              - 'frontend/**'
```

Comment:
Is this intended to be part of this PR?

Reply:
Yes - we need to revert the changes to deploy frontend-react

Reply:
we need to revert this _full stop_ or _we need to revert this as part of this PR_ ?

Reply:
The line is incorrect and will not work properly...this needs to be reverted

Reply:
Discussed in person